mesonbuild/dependencies/ui.py

```diff
@@ -18,7 +18,9 @@
 import os
 import re
 import subprocess
+import typing as T
 from collections import OrderedDict
+import typing as T
```

Comment:
__[Flake8]__ redefinition of unused 'T' from line 21 ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6443#issue-5055574))

<details>

| Rule |
| - |
| F811 |

</details>
<sub>posted by Sider</sub>